.github/workflows/release.yml

```diff
@@ -47,6 +47,9 @@ jobs:
         with:
           ruby-version: 'jruby-9.4.2.0'
           bundler-cache: false
+      - name: Install Bundler 2.6.9
+        if: ${{ inputs.with_ruby == 'true' }}
```

Comment:
This workflow does not accept the `with_ruby` parameter, so perhaps we install the bundler by default?